text/0045-nft-deposits-asset-hub.md

```diff
@@ -116,6 +116,28 @@ This adjustment would result in the following DOT and KSM deposit values:
 | `metadataDepositBase`     | 0.002 DOT                  | 0.0006709666617 KSM      |
 | `attributeDepositBase`    | 0.002 DOT                  | 0.000666666666 KSM       |
 
+### Short- and Long-Term Plans
+
+The plan presented above is recommended as an immediate step to make Polkadot a more attractive
+place to launch NFTs, although one would note that a forty fold reduction in the Item Deposit is
+just as arbitrary as the value it was replacing. As explained earlier, this is meant as a subsidy to
+gain more momentum for NFTs on Polkadot.
+
+In the long term, an implementation should account for what should happen to the deposit rates
+assuming that the subsidy is successful and attracts a lot of deployments. Many options are
+discussed in the [Addendum](#addendum).
+
+The deposit should be calculated as a function of the number of existing collections with maximum
+DOT and stablecoin values limiting the amount. With asset rates available via the Asset Conversion
+pallet, the system could take the lower value required. A sigmoid curve would make sense for this
+application to avoid sudden rate changes, as in:
+
+$$ \frac{\mathrm{min(DOT deposit, stable deposit)} }{\mathrm{1 + e^{a - b * x}} }$$
```

Comment:
```suggestion
$$ startdeposit + \frac{\mathrm{min(DOTdeposit, stabledeposit) - startdeposit} }{\mathrm{1 + e^{b * a - b * x}} }$$
```

Reply:
Ah, I see what you are doing. But you don't need `b * a` because its evaluation is implicit in the value of `a`.

Reply:
Okay but then the code implementor has to know that a = x0 * b, where x0 is the threshold level of collections or items.